Fix Vale warnings

ptgott opened the pull request:
Fix Vale warnings across several guides within the `docs/pages/admin-guides` section.

ptgott committed:
Fix Vale warnings

Fix Vale warnings across several guides within the
`docs/pages/admin-guides` section.

flyinghermit commented on docs/pages/admin-guides/access-controls/access-requests/oss-role-requests.mdx:
```diff
@@ -20,7 +20,7 @@ users can request access to. In this example, we will define two roles:
 There is no role for request approvers, because request approval rules can only
 be configured for Teleport Enterprise. In Teleport Community Edition, approvals must
-be performed by running `tctl` on the Auth Server.
+be performed by running `tctl` on an Auth Service instance.
```
is mention of "instance" necessary here? we do not mention it below.

ptgott committed:
Fix Vale issues in 36 docs guides

Fix Vale warnings

Fix Vale warnings across several guides within the
`docs/pages/admin-guides` section.

This includes removing the `teleport-cluster` migration guide, which
includes some Vale issues. This was an overdue TODO item.

ptgott committed:
Merge branch 'master' into paul.gottschling/2024-12-27-vale
